API.md

```diff
@@ -13,9 +13,15 @@
 <dt><a href="#getViewportSize">getViewportSize(deviceName)</a> ⇒ <code><a href="#DeviceViewportSize">DeviceViewportSize</a></code></dt>
 <dd><p>Gets the viewport size for the specified device.</p>
 </dd>
+<dt><a href="#isMaximized">async isMaximized(pageTitle)</a> ⇒ <code>boolean</code></dt>
+<dd><p>Checks if the specified browser window is maximized.</p>
+</dd>
 <dt><a href="#isValidDeviceName">isValidDeviceName(inputString)</a> ⇒ <code>boolean</code></dt>
 <dd><p>Checks if the provided string is a valid device name contained in the screen size database.</p>
 </dd>
+<dt><a href="#maximize">async maximize(pageTitle)</a></dt>
```

Comment:
Trouble is that in order to do the second resize we need a new client size info after the first resize or window restoring. In that case `testcafe` must be able to execute the specified code or resend client information while the corresponding action is running. There is no such mechanism in `testcafe` yet. But I've implemented a mechanism, that allows to execute code in browser on idle page, and that means we can do the things just after the browser has started.

The problem with restoring down that because we can't do it on resize we need to do it even if resize is not used at all. I think it's not nice. What we have discussed with @inikulin is the:
```
on openBrowser:
    open the browser
    if browser is not maximized
        return
    get client area
    resize or restore down
    get new client area
    calculate corrections using old and client area size
    //addition
   restore window to its maximized state //to play nice

on resize:
    if browser is maximized
        apply calculated corrections

    resize the browser
```